samples/apps/copilot-chat-app/webapi/Storage/ChatMessageRepository.cs

```diff
@@ -0,0 +1,25 @@
+﻿using SKWebApi.Skills;
+
+namespace SKWebApi.Storage;
+
+public class ChatMessageRepository : Repository<ChatMessage>
+{
+    public ChatMessageRepository(IStorageContext<ChatMessage> storageContext)
+        : base(storageContext)
+    { }
+
+    public Task<IEnumerable<ChatMessage>> FindByChatId(string chatId)
```

Comment:
All methods returning "Task" should be suffixed with Async where possible (e.g. `FindByChatIdAsync`).

Reply:
Same comment for all classes in /Storage